packages/studio/frontend/src/Pages/Overview.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { Title, Grid, Col } from '@tremor/react'
+
+import SeriesTypeBarList from '../BarLists/SeriesTypeBarList'
+
+export default function App() {
+  return (
+    <div className="bg-slate-50 p-6 sm:p-10 h-full">
```

Comment:
```suggestion
    <div className="p-6 sm:p-10 h-full">
```

Just causes a difference in the page background half way down the page when the div stops.